apps/ledger-live-mobile/src/screens/WalletCentricSections/Allocations.tsx

```diff
@@ -88,7 +91,7 @@ const Allocations = () => {
         ticker: t("common.others"),
         color: colors.neutral.c70,
         type: "CryptoCurrency",
-      },
+      } as CryptoCurrency,
```

Comment:
no way to give the type `DistributionItem` to `othersAllocations` instead of using `as` here ?

```ts
const othersAllocations: DistributionItem = ...
```

Reply:
fixed